tests/DependencyInjection/Fixtures/ServiceA.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Tests\DependencyInjection\Fixtures;
+
+class ServiceA implements ServiceInterface
+{
+    public function getName(): string
+    {
+        return 'service_a';
+    }
+
+    public function getDefault(string $mode = self::MODE_FOO): ?string
```

Comment:
Maybe we can use this test to check different argument representations. Ex: without type, without default value, PHP8 attributes, variadic arguments, etc.)

Reply:
I test it here [tests/Util/ClassSource/ClassMethodTest.php#L62](https://github.com/WedgeSama/maker-bundle/blob/feature/make-decorator/tests/Util/ClassSource/ClassMethodTest.php#L62)
Is it enough?